### DISCUSSION TITLE
[Feature Request] How to hide method don't want show in API document

### DISCUSSION THREAD

**Discussion by chuongmep** [dotnet/docfx#8504] [General]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**
 I want to hide some class and method, how to can do that ? 

**Describe alternatives you've considered**

Example I have a class with name ABC and I want to hide it.

**Additional context**

**Comment by yufeih** [comment]
This is achievable through the [filter config](https://dotnet.github.io/docfx/docs/dotnet-api-docs.html#custom-with-filter-rules).
